.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           -G "Unix Makefiles" \
           -DCMAKE_BUILD_TYPE=Debug \
           -DBUILD_CLONE_SUBMODULES=ON \
+          -DUNITTEST=1 \
```

Comment:
Does it make sense to do coverity and unit test build in the CI separately then?

it seems like it would be really hard to make a change that breaks the unit test build but not the unit test one? But just wanted to ask about it